app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
+}
```

Comment:
The variable `collection_of_coins` is categorized as mutable, but it is a set, which is considered mutable in Python. However, since sets themselves cannot contain mutable elements, the categorization is somewhat ambiguous. If the task requires strict categorization, consider clarifying the criteria for mutability.